portal/views/patients.py

```diff
@@ -46,23 +51,33 @@ def patients_root():
         ).join(UserOrganization).filter(
             and_(UserOrganization.user_id==User.id,
                  UserOrganization.organization_id.in_(org_list)))
+    leaf_organizations = None
+    try:         
+        leaf_organizations = user.leaf_organizations()
+    except ValueError:
+         current_app.logger.debug("In patients list: error retrieving leaf organizations for user %s", str(user.id))
```

Comment:
Okay, I'd rather fix the source - it should return None w/o orgs in the tree, not raise an error.  Care to make that change, or should I?